playbooks/production/roles/common/tasks/main.yml

```diff
@@ -0,0 +1,95 @@
+- name: Update apt cache
+  apt: update_cache=yes cache_valid_time=3600
+
+- name: Install packages
+  apt: name={{ item }}
+  with_items:
+    - git
+    - docker.io
+    - docker-compose
+    - python3-pip
+    - python3-venv
+    - curl
+    - libssl-dev
+    - pkg-config
+
+#- name: Add to fstab
+#  ansible.builtin.lineinfile:
+#    path: /etc/fstab
+#    line: "UUID=64d72737-f1de-4f67-a22a-f693a79f228f /data ext4 defaults,nofail 0 2"
+#    state: present
+
+- name: Mount the disk
+  ansible.posix.mount:
+    path=/data src=UUID=64d72737-f1de-4f67-a22a-f693a79f228f fstype=ext4 state=mounted
```

Comment:
I believe this is a volume in Azure. I think this UUID should be retrieved first with `k8s_info` task, exported as a fact and only then used here (I'll look up similar task as an example later - ping me if I forget to do so :sweat_smile:)

Reply:
It is legacy block device i want to reuse, unfortunately this is old storage stuff i have to migrate first, but then system will be cleaned up from legacy things and recipe will change. 
Main purpose of this initial recipe how to image system fast and bring up on new distro with minimum downtime (and reverse if something doesnt work).
Then, final recipe after i will finish storage cleanup will be done.